src/index.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-import { Provider } from 'react-redux';
+import { Provider, compose } from 'react-redux';
```

Comment:
@elimisteve I get exactly the same. Even though i have changed the setup for Redux dev tools according to how it should be done after some updates. They have changed it a bit. 

This was not a problem using webpack, actually if you use webpack with hot module replacement , then Redux dev tools can hot reload Reducers. I guess we can skip using Redux dev tools until we have a good setup with webpack?